egov-persister/service-request-persister.yml

```diff
@@ -35,7 +35,7 @@ serviceMaps:
 
             - jsonPath: $.ServiceDefinition.auditDetails.lastModifiedTime
 
-            - jsonPath: $.ServiceDefinition.additionalDetails
+            - jsonPath: $.ServiceDefinition.additionalFields
```

Comment:
@vishal-egov did we already migrate to the version which has additionalFields instead of additionalDetails in checklist definition?

Reply:
yeah we can do this change as we will new build